src/components/Footer.tsx

```diff
@@ -1,5 +1,191 @@
 import React from 'react';
-
+import logo from '../image/logo-tnj-footer.png';
+import plane from '../image/paper-plane.svg';
+import Input from './Input';
+import logoSaleNoti from '../image/logoSaleNoti.png';
+import house from '../image/house.svg';
+import clock from '../image/clock.svg';
+import phone from '../image/phone.svg';
+import mail from '../image/mail.svg';
+import { Link } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
```

Comment:
```suggestion
import { Link } from 'react-router-dom';
import { useTranslation } from 'react-i18next';

```
Thêm khoảng trống phía sau